packages/node-cli/src/commands/CreateProjectCommandGenerator.js

```diff
@@ -275,7 +275,7 @@ module.exports = class CreateProjectCommandGenerator extends BaseCommandGenerato
 
 				const operationResult = await this._sdkExecutor.execute(executionContextCreateProject);
 
-				if (SDKOperationResultUtils.hasErrors(operationResult)) {
+				if (operationResult.status === SDKOperationResultUtils.ERROR) {
```

Comment:
I prefer the hasErrors(operationResult) function than the straight comparison to the ERROR. And if so, maybe it would be better to use it like SDKOperationResultUtils.STATUS.ERROR

Reply:
I was having an error in that function, so, because we deleted a similar function in ActionResultUtils, I decided to do the same here.... 
Also the method now would be better defined as, isStatusError
I like your second idea, I will implement it.

Reply:
I think it's not bad, quite readable, the function will only have 1 line, not sure it's worth creating it